addons/account_payment_term/tests/test_payment_term.py

```diff
@@ -0,0 +1,53 @@
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+from odoo import Command, fields
+from odoo.tests.common import Form
+
+
+@tagged('post_install', '-at_install')
+class TestAccountPaymentTerms(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.invoice = cls.init_invoice('out_refund', products=cls.product_a + cls.product_b)
+
+        cls.pay_term_a = cls.env['account.payment.term'].create({
+            'name': "basic case",
+            'line_ids': [
+                Command.create({
+                    'value': 'percent',
+                    'value_amount': 100,
+                    'nb_days': 30,
+                    'delay_type': 'days_end_of_month_on_the',
+                    'days_next_month': 10,
+                }),
+            ],
+        })
+        cls.pay_term_b = cls.env['account.payment.term'].create({
+            'name': "special case",
+            'line_ids': [
+                Command.create({
+                    'value': 'percent',
+                    'value_amount': 100,
+                    'nb_days': 30,
+                    'delay_type': 'days_end_of_month_on_the',
+                    'days_next_month': 31,
+                }),
+            ],
+        })
+
+    def test_payment_term_days_end_of_month_on_the(self):
+        with Form(self.invoice) as basic_case:
```

Comment:
Could you please explain in the docstring what's the purpose of the test?
As I review, I know what is `days_end_of_month_on_the` but I'm scared that later, someone will be "They forgot to finish their function name, so dumb" :grimacing: